Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -0,0 +1,541 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Matteo Beccati <matteo@beccati.com>
+ */
+class Php4ConstructorFixerTest extends AbstractFixerTestBase
+{
+    public function testNamespaces()
```

Comment:
@stof I had started doing that, but the test file became almost unreadable, so I've reverted the changes. I think that having $expected and $input helps readability.